src/platform/elements/date-picker/DatePickerInput.ts

```diff
@@ -45,20 +46,28 @@ export class NovoDatePickerInputElement implements OnInit, ControlValueAccessor
   @Input() placeholder: string;
   @Input() maskOptions: any;
   @Input() format: string;
+  @Input() userDefinedFormat: boolean;
```

Comment:
Sorry, I just had a thought, could you still use format but do a regex on the `this.format` to check if its `dd/dd/dddd` to use the mask vs not?